app/controllers/items_controller.rb

```diff
@@ -0,0 +1,54 @@
+class ItemsController < ApplicationController
```

Comment:
These are APIs controller, so both controllers will be in namespace /api/v1.
Path will be in `app/controllers/api/v1/items_controller.rb`
`app/controllers/api/v1/items_controller.rb`